histomicstk/segmentation/label/delete_overlap.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+from .condense import condense
+from .delete import delete
+
+
+def delete_overlap(im_label, overlap_info):
+    """
+    Deletes overlapping regions from an image label based on overlap information and tile size.
+
+    Args:
+        im_label (ndarray): Image label represented as a NumPy array.
+        overlap_info (dict): Dictionary containing overlap information.
+                            It should have the following keys:
+                            'left', 'right', 'top', and 'bottom',
+                            each specifying the overlap amount in pixels.
+
+    Returns:
+        ndarray: Image label with overlapping regions deleted.
+    Note:
+        This function assumes the necessary imports, such as `np`, are already present.
+
+    """
+
+    # Compute the half of the overlap values
+    left_overlap = max(overlap_info['left'] // 2, 0)
```

Comment:
I don't think the overlap info can ever have negative values.  Is the `max` necessary?